src/rmrk2.0.0/tools/consolidator/interactions/buy.ts

```diff
@@ -175,7 +175,7 @@ const validate = (
       throw new Error(
         `[${OP_TYPES.BUY}] Attempting to buy not-for-sale NFT ${buyEntity.id}`
       );
-    case nft.transferable === 0 || nft.transferable >= remark.block:
+    case nft.transferable === 0 || nft.transferable >= remark.block || (((nft.block - nft.transferable) < remark.block) && (nft.transferable < 0)):
```

Comment:
I don't understand the need for this last part of condition `&& (nft.transferable < 0)`

Reply:
If there is no check whether the transferable input is negative it would always evaluate to true right? Or did i make a mistake there

Reply:
How about this way https://github.com/rmrk-team/rmrk-tools/pull/100#discussion_r847760188